src/routes/PostDetail/Comments/Comments.connector.js

```diff
@@ -2,16 +2,22 @@ import { connect } from 'react-redux'
 import getMe from 'store/selectors/getMe'
 import { isEmpty } from 'lodash/fp'
 import fetchComments from 'store/actions/fetchComments'
+import { findCommentId } from 'util/navigation'
```

Comment:
not used? unless we want to load it here and pass down, though again should be in routeParams I think and you could use getRouteParam